spec/services/preserved_object_handler_check_exist_spec.rb

```diff
@@ -386,6 +376,38 @@
         it_behaves_like 'unexpected version with validation', :check_existence, 1, PreservedCopy::EXPECTED_VERS_NOT_FOUND_ON_STORAGE_STATUS
       end
 
+      context 'PreservedCopy already has a status other than OK_STATUS' do
+        it_behaves_like 'PreservedCopy already has a status other than OK_STATUS, and incoming_version == pc.version', :check_existence
+
+        it_behaves_like 'PreservedCopy already has a status other than OK_STATUS, and incoming_version < pc.version', :check_existence
+
+        context 'incoming_version > db version' do
+          let(:incoming_version) { pc.version + 1 }
+
+          it 'had OK_STATUS, version increased, should still have OK_STATUS' do
+            pc.status = PreservedCopy::OK_STATUS
+            pc.save!
+            allow(po_handler).to receive(:moab_validation_errors).and_return([])
+            po_handler.check_existence
+            expect(pc.reload.status).to eq PreservedCopy::OK_STATUS
+          end
+          it 'had INVALID_MOAB_STATUS, was remediated, should now have OK_STATUS' do
+            pc.status = PreservedCopy::INVALID_MOAB_STATUS
+            pc.save!
+            allow(po_handler).to receive(:moab_validation_errors).and_return([])
+            po_handler.check_existence
+            expect(pc.reload.status).to eq PreservedCopy::OK_STATUS
+          end
+          it 'had EXPECTED_VERS_NOT_FOUND_ON_STORAGE_STATUS, seems to have an acceptable version now' do
```

Comment:
"seems to have" ?   hahaha - it's a test - I think we could be assertive here.  Mostly added this comment to amuse myself, not because I think changing the text is important.

Reply:
"gee, sure looks like it has an acceptable version now, but really who are we to judge what's an acceptable version?"